src/scenes/home/families/facts/facts.css

```diff
@@ -1,26 +1,54 @@
-
 .facts {
-  display: 'flex';
-  background-color: '#E2E2E2';
-  align-items: 'center';
-  padding: '96px';
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 50px 0;
+}
+
+@media screen and (max-width: 1024px) {
+  .facts {
+    display: block;
+  }
 }
-.factsHeading {
-  font-size: '50px';
-  text-align: 'center';
+
+.factsHeading h2 {
+  font-size: 50px;
```

Comment:
We're doing our best to avoid px sizing fonts. You'll see on `src/index.css` that we have a REM system that changes on break-points dynamically. I know that you we're aiming for the mock-up, and you matched it perfectly.

Try to balance what you've already got while choosing responsive text-sizes that make sense. This one in particular beats our h1 by 10 px on large screens - so it goes against our styles pretty heavily.

Reply:
At the same token, perhaps we should up-size our text scale. Please feel free to start a debate on that in Slack. Sometimes I feel like the text on our site is too small.

If we're to scale our fonts up, we'll likely have some things break, but it'll be a good opportunity to wipe our project clean of all px sized fonts, and add a notch to the sustainable stylesheets belt.

Reply:
Cleaned up the `px` font sizes in `goals.css` too.